roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -34,6 +42,8 @@ def __init__(self, execute_module=None, task_vars=None, tmp=None):
         self._execute_module = execute_module
         self.task_vars = task_vars or {}
         self.tmp = tmp
+        # set True when the check makes a change to the host so it can be reported to the user:
```

Comment:
s/set True/set to True/ ?

s/so it can be reported to the user// ? "reporting to the user" is kind of optional, we don't need to make promises in this comment, but I'll leave it up to you, I'm fine either way.